homeassistant/components/image/__init__.py

```diff
@@ -120,6 +125,7 @@ class ImageEntity(Entity):
 
     def __init__(self) -> None:
```

Comment:
```suggestion
    def __init__(self, hass: HomeAssistant) -> None:
```

Reply:
I have updated the code, all existing code was updated to pass `hass`.